package.json

```diff
@@ -149,7 +149,7 @@
     "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
     "build:docs": "node scripts/generateDocs.mjs",
     "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
-    "build:sass:component": "sass --no-source-map src/style.scss:src/style.css",
+    "build:sass:component": "for file in src/*.scss; do npx sass --no-source-map \"$file:${file%.scss}.css\"; done",
```

Comment:
This is interesting and clever, make sure to open a ticket in the generator to migrate this there! Nice work.

Reply:
I saw the Discussion on updating this. I'd like to understand this more on why we now need a `for` loop and the use of `npx` over a local dependency? 

https://github.com/orgs/AlaskaAirlines/discussions/557#discussioncomment-9799161

Reply:
Maybe we don't have an inherent need for this in all of our components. This script was my solution to the issue where the `color.scss` and `tokens.scss` file would not compile.